test/system/runScenario1.js

```diff
@@ -0,0 +1,13 @@
+const runScenario1 = (trader1) => {
+  return new Promise((resolve) => {
+    trader1.placeOrder('S')
+      .then(() => trader1.placeOrder('B'))
+      .then(() => trader1.placeOrder('S'))
+      .then(() => trader1.placeOrder('B'))
+      .then(() => resolve(true))
```

Comment:
Scenarios need to be independent and isolated and must leave no side-effects like open orders. The last step here should be to cancel market orders